paymentsheet/src/main/java/com/stripe/android/paymentsheet/LinkHandler.kt

```diff
@@ -69,23 +54,12 @@ internal class LinkHandler @Inject constructor(
     val isLinkEnabled: StateFlow<Boolean?> = _isLinkEnabled
 
     private val _linkConfiguration = MutableStateFlow<LinkConfiguration?>(null)
-    private val linkConfiguration: StateFlow<LinkConfiguration?> = _linkConfiguration.asStateFlow()
+    val linkConfiguration: StateFlow<LinkConfiguration?> = _linkConfiguration.asStateFlow()
```

Comment:
How come this needs to be exposed now?

Reply:
`linkConfiguration` used to only be used internally. Since we moved the Link native/web flow into a definition, we need access to the `linkConfiguration` in `PaymentSheetViewModel`. Once we move `LinkInlineSignup`, we can rid of this class entirely and just handle `linkConfiguration` in `PaymentSheetViewModel`.